component-samples/demo/aio/docker-compose.yml

```diff
@@ -5,6 +5,24 @@ version: "3.3"
 
 services:
 
+  fdo-db:
+    image: mariadb:11.4.2
+    container_name: fdo-db
+    restart: always
+    ports:
+      - 3306:3306
+    environment:
+      - MARIADB_ROOT_PASSWORD_FILE=/run/secrets/db_password.txt
+      - MARIADB_DATABASE=emdb
+    secrets:
+      - ca-cert.pem
+      - server-cert.pem
+      - server-key.pem
+      - db_password.txt
+    volumes:
+      - ./app-data:/var/lib/mysql
+      - ./custom:/etc/mysql/conf.d
```

Comment:
Wouldn't this be started with https://github.com/fido-device-onboard/test-fidoiot/blob/1ab5a3c57c76c61a3d8c405eeb039835ddb5d979/priTests/src/test/java/org/fidoalliance.fdo/test/PriSmokeTest.java#L74?

https://github.com/fido-device-onboard/pri-fidoiot/blob/master/component-samples/demo/db/docker-compose.yml

Reply:
Perhaps it is the order that is the problem - the `[PriSmokeTest.java](https://github.com/fido-device-onboard/test-fidoiot/blob/1ab5a3c57c76c61a3d8c405eeb039835ddb5d979/priTests/src/test/java/org/fidoalliance.fdo/test/PriSmokeTest.java#L73-L75) starts aio before the db...